src/models/plateau.rs

```diff
@@ -0,0 +1,169 @@
+use crate::models::{piece, point};
+
+use piece::Piece;
+use point::Point;
+use std::fmt;
+
+const PLAYER1: char = '0';
```

Comment:
This needs to be an 'O' (oh) and not a '0' (zero)

Reply:
Good Catch!